mappings/net/minecraft/particle/ParticleEffect.mapping

```diff
@@ -1,7 +1,9 @@
 CLASS net/minecraft/unmapped/C_nqucohct net/minecraft/particle/ParticleEffect
 	METHOD m_dkcpurjj getType ()Lnet/minecraft/unmapped/C_qlzoaopu;
 	METHOD m_wtvchkob asString (Lnet/minecraft/unmapped/C_vtbxyypo$C_etmlgbig;)Ljava/lang/String;
+		ARG 1 provider
```

Comment:
What kind of provider is this?

Reply:
I labeled this as just a generic provider, as every implementation of this in the code doesn't appear to use it.